### PR TITLE
Cleanup/Name change

### PR DESCRIPTION
**Pull request by jfeiber**
Commented out example code in server.go.

Renamed some stuff.

@joonhyongcho @MJAlperin23

### CODE REVIEW

File: db/migrations/20161013155400_buzzerdb.sql [hunk 1]
```diff
@@ -1,23 +1,13 @@
 -- +goose Up
-CREATE TABLE Device (
```
Comment (MJAlperin23) [inline db/migrations/20161013155400_buzzerdb.sql:2]: I thought we might want to leave this for now just to test stuff but if you think we are good that works for me

File: db/migrations/20161013155400_buzzerdb.sql [hunk 2]
```diff
@@ -59,7 +49,6 @@ CREATE TABLE WebAppUser (
 
 
 -- +goose Down
-DROP TABLE Device;
 DROP TABLE Buzzer;
 DROP TABLE HistoricalParties;
 DROP TABLE WebAppUser;
```
Comment (MJAlperin23) [inline db/migrations/20161013155400_buzzerdb.sql]: also needs to change to new name

File: db/migrations/20161013155400_buzzerdb.sql [hunk 3]
```diff
@@ -59,7 +49,6 @@ CREATE TABLE WebAppUser (
 
 
 -- +goose Down
-DROP TABLE Device;
 DROP TABLE Buzzer;
 DROP TABLE HistoricalParties;
```
Comment (MJAlperin23) [inline db/migrations/20161013155400_buzzerdb.sql]: needs to change to new name

File: models.go [hunk 4]
```diff
@@ -4,25 +4,14 @@ import (
...
-    restaurantID int
+    id int
     name string `gorm:"size:99; not null"`
     dateCreated time.Time `gorm:"not null"`
 }
 
 type ActiveParty struct {
-    activePartyID int
+    id int
     restaurant Restaurant
```
Comment (MJAlperin23) [inline models.go:15]: are we sure these foreign keys will work now that you changed restaurantID to just id which they all have

Comment (jfeiber) [reply models.go:15]: They won't I forgot to commit some more changes to the models file before PRing, it's fixed now

File: db/migrations/20161013155400_buzzerdb.sql (hunk 1 above)
Comment (jfeiber) [reply db/migrations/20161013155400_buzzerdb.sql:2]: I think we're fine, probably better to test stuff using our actual tables anyway.

**Comment by jfeiber** [comment]
Replaced by https://github.com/jfeiber/buzzer/pull/6

**Pull request closed without merging**